baseapp-e2e/README.md

```diff
@@ -0,0 +1,45 @@
+# BaseApp E2E
+
+## Usage
+
+This project includes utilities for database initialization and seeding when performing End-To-End tests
+
+### E2E Enpoints
```

Comment:
Can we just add a little example of a the format of the objects that we need to send to the endpoint or a link to the documentations, pls?